Added release workflow

johannwagner opened the pull request:
Adding a release workflow that builds a various amount of binariers for different applications and attaches it to a created release.

annikahannig reviewed (changes requested):
Thank you! This is really looking promising.

The release build however would right now not include the correct build / version.

In the makefile this is done by providing these flags to the compiler:


` -X github.com/b3scale/b3scale/pkg/config.Version=$(VERSION) -X github.com/b3scale/b3scale/pkg/config.Build=$(BUILD)`

johannwagner commented:
I added the missing ldflags from the `github` context. Note: I added a long commit sha instead of a short one, because thats what is available in the CI easily.

annikahannig commented:
I don't like it.
Let's use this anyhow. I do not care enough.
